src/methods.jl

```diff
@@ -38,3 +38,44 @@ julia> bias_subtraction(frame, bias)
 * [`bias_subtraction!`](@ref)
 """
 bias_subtraction(frame::AbstractArray, bias_frame::AbstractArray) = bias_subtraction!(deepcopy(frame), bias_frame)
+
+
+#=
+***Note to future developer***
```

Comment:
I'd say that all comments are for future developers, including the future you. Just remove this line :wink: